### ISSUE TITLE
Function ENVS are not injected properly when running a function locally via kyma CLI

### ISSUE THREAD

**Issue by kwiatekus** [kyma-project/cli#1112]
**Description**

Function ENVS are not properly injected when running a function in a local docker runtime

**Expected result**

Function ENVS should be properly injected and resolvable in the handler when running a function locally via kyma CLI

**Actual result**

Function ENVS are not injected properly when running a function locally via kyma CLI

**Steps to reproduce**

1. Run `kyma init function` in an empty folder
2. Define `config.yaml` and `handler.js` as attached
3. Run function locally via `kyma run function --hot-deploy`
4. Observe that `process.env[]` are resolved to undefined 

Same function works fine when deployed to kyma runtime.

**Attachments**

 - sample manifest
 
 ```
name: my-function
namespace: dev
runtime: nodejs14
source:
  sourceType: inline
  sourceHandlerName: handler.js
  depsHandlerName: package.json
env:
  - name: MICROSERVICE_SERVICE_HOST
    value: localhost
  - name: MICROSERVICE_SERVICE_PORT
    value: '3000'
 ```
  - sample handler
  
```
module.exports = {
    main: function (event, context) {
        console.log("Host",process.env['MICROSERVICE_SERVICE_HOST'])
        console.log("Port",process.env['MICROSERVICE_SERVICE_PORT'])
        return 'Hello Serverless'
    }
} 
```

**Comment by pPrecel** [comment]
Good idea. But what with envs that should get value from the ConfigMap of the Secret? Do we want to support it now? Should we inject generated value ( for example `MOCKED_VALUE` ) or get the right value from the cluster if communication is possible?

Example [here](https://kyma-project.io/docs/kyma/latest/05-technical-reference/svls-06-function-configuration-file/#specification-for-an-inline-function):

```
env:
    - valueFrom:
        configMapKeyRef:
          Name: configmap1
          Key: token-field
    - valueFrom:
        secretKeyRef:
          Name: secret1
          Key: token-field
```

**Comment by kwiatekus** [comment]
Injections from `configMapKeyRef` and `secretKeyRef` are not in scope for this case.
I imagine users run functions locally (in hot deploy mode) to test the code. For that purpose users can inject ENVS using simple values. Once the code proves to work users should deploy to kyma runtime and only then they could start thinking of better env management via secrets or config maps. 
Lets document that in case of locally run functions only simple ENV injection is supported

**Comment by kwiatekus** [comment]
Fixed in kyma CLI [v2.0](https://github.com/kyma-project/cli/releases/tag/2.0.0)
